data_processing/meteo/task_functions.py

```diff
@@ -0,0 +1,171 @@
+from datagouvfr_data_pipelines.config import (
+    AIRFLOW_DAG_HOME,
+    AIRFLOW_DAG_TMP,
+    DATAGOUV_SECRET_API_KEY,
+    AIRFLOW_ENV,
+    MINIO_URL,
+    SECRET_MINIO_DATA_PIPELINE_USER,
+    SECRET_MINIO_DATA_PIPELINE_PASSWORD,
+)
+from datagouvfr_data_pipelines.utils.datagouv import (
+    post_remote_resource,
+    DATAGOUV_URL
+)
+from datagouvfr_data_pipelines.utils.mattermost import send_message
+from datagouvfr_data_pipelines.utils.minio import send_files, get_all_files_from_parent_folder
+import ftplib
+import os
+import json
+from datetime import datetime
+import re
+import requests
+
+DAG_FOLDER = "datagouvfr_data_pipelines/data_processing/"
+DATADIR = f"{AIRFLOW_DAG_TMP}meteo/data"
+
+
+def list_ftp_files_recursive(ftp, path='', base_path=''):
```

Comment:
You may move this to a utils.ftp?

Reply:
This has been discussed with @HAEKADI, FTP processes can differ so we don't really know if that's a good idea or not... TBD